cloudbuild_public.yaml

```diff
@@ -0,0 +1,70 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: 'bash'
+  args: [
+    '-c',
+    'docker pull us.gcr.io/$PROJECT_ID/$REPO_NAME:nightly || true',
+  ]
+- name: 'gcr.io/cloud-builders/docker'
+  args: 
+    [
+      'build',
+      '-t', 'us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA',
+      '-t', 'us.gcr.io/$PROJECT_ID/$REPO_NAME:nightly',
+      '--cache-from', 'us.gcr.io/$PROJECT_ID/$REPO_NAME:nightly',
+      '--cache-from', 'us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA',
+      '.'
+    ]
+# CDC tests
+- name: 'gcr.io/$PROJECT_ID/docker-compose'
+  args:
+    - '-f'
+    - 'docker-compose.gcb.yml'
+    - run
+    - '--rm'
+    - cdc-test
+# Clean up after tests
+- name: 'gcr.io/$PROJECT_ID/docker-compose'
+  args:
+    - '-f'
+    - 'docker-compose.gcb.yml'
+    - down
+    - '--rmi'
+    - 'local'
+    - '-v'
+    - '--remove-orphans'
```

Comment:
I don't think we need to be this aggressive on clean up. 😀  (not objecting, just a bit surprised at the level of scrubbing)

Reply:
removed